src/components/MedicalServiceMasterPanel.js

```diff
@@ -152,6 +181,28 @@ const mapStateToProps = (state) => ({
   savedServiceCode: state.medical?.medicalService?.code,
 });
 
+const validationSchema = yup.object({
+  serviceCode: yup.string().required('Service code is required'),
+  serviceName: yup.string().required('Service name is required'),
+  serviceType: yup.string().required('Service type is required'),
+  serviceCareType: yup.string().required('asdad')
```

Comment:
Not a error message.

Reply:
Yes, I am aware of those problems, it is only a proof of concept at this moment, further development is needed. I've left this PR as a reference.